kano_greeter/last_user.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+import os
+
+LAST_USER_PATH = '/var/lib/lightdm/.cache/kano-greeter/'
+LAST_USER_FILE = 'last-user'
```

Comment:
True, will change that.

I wasn't sure about the path to use. /var/lib/lightdm/.cache/ is what lightdm-gtk-greeter uses, but since kano-greeter comes as a separate project, maybe /var/lib/kano-greeter is more appropriate?